style.css

```diff
@@ -1,4 +1,795 @@
+  position: relative;
+}
+
+.tnw-logo-up {
+  display: none;
+}
+
+.main-up-text {
+  position: absolute;
+  bottom: 0;
+  color: white;
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 10%, rgba(0, 0, 0, 1) 80%);
+  width: 100%;
+}
+
+.main-up-text * {
+  font-family: "Avalon", sans-serif;
+  padding-left: 5px;
+  font-size: 1em;
+}
+
+.main-items-up h2 {
+  font-size: 1em;
+}
+
+.main-up-text h3 {
+  font-size: 14px;
+  font-family: "ShentoxW01-UltraLight", sans-serif;
+}
+
+.main-up-text p {
+  font-size: 12px;
+  color: #aaa;
+  font-family: "ARS+Maquette+Web+One", sans-serif;
+}
+
+.first-grid {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.title-first {
+  width: 95%;
+  margin: 60px auto 30px auto;
+}
+
+.title-first h2 {
+  font-size: 28px;
+  color: #f42;
+  font-weight: 700;
+  font-family: "Avalon", sans-serif;
+}
+
+.news-container {
+  display: grid;
+  gap: 14px;
+  grid-template-rows: repeat(8, minmax(auto, 1fr));
+  height: auto;
+  width: 95%;
+  margin: 0 auto 60px auto;
+}
+
+.news-items {
+  height: 100%;
+}
+
+.news-items h2 {
+  font-size: 16px;
+  font-family: "ARS+Maquette+Web+One", sans-serif;
+  color: #000;
+}
+
+.news-items p {
+  color: #a9a9a9;
+  line-height: 2em;
+  font-size: 13px;
+  font-family: "ARS+Maquette+Web+One", sans-serif;
+}
+
+.news-container img {
+  height: 80px;
+  width: auto;
+  float: left;
+  padding-right: 18px;
+}
+
+.funding-title-div {
+  width: 95%;
+  margin: 0 auto 30px auto;
+  color: #f42;
+  line-height: 2em;
+}
+
+.funding-header {
+  font-size: 28px;
+  font-family: "Avalon", sans-serif;
+  color: #f42;
+}
+
+.funding-anchors {
+  padding: 0;
+}
+
+.funding-grid-container {
+  display: grid;
+  width: 95%;
+  margin: 0 auto;
+  gap: 20px;
+  grid-template-rows: repeat(4, auto);
+}
+
+.funding-grid-items {
+  position: relative;
+}
+
+.orange p,
+.lightorange p {
+  padding-left: 100px;
+}
+
+.smartlogo {
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  height: 70px;
+  padding-top: 20px;
+}
+
+.orange {
+  background-color: #f42;
+  height: 45px;
+}
+
+.orange p {
+  color: white;
+  font-size: 18px;
+  font-weight: 700;
+  padding-top: 11px;
+  font-family: "Avalon", sans-serif;
+}
+
+.lightorange {
+  background-color: #ffebe7;
+  height: 59px;
+}
+
+.lightorange p {
+  font-size: 18px;
+  font-weight: 700;
+  padding-top: 14px;
+  font-family: "Avalon", sans-serif;
+  margin: 0;
+}
+
+.beige1 {
+  background-color: #fffaf8;
+  height: 80px;
+  border-bottom: 1px solid #f42;
+}
+
+.beige1 p,
+.beige2 p {
+  font-size: 13px;
+  padding-left: 15px;
+  padding-top: 13px;
+  font-family: "ARS+Maquette+Web+One", sans-serif;
+  color: #aaa;
+  margin: 0;
+}
+
+.funding-anchors a {
+  font-size: 14px;
+  color: #f42;
+  font-family: "ARS+Maquette+Web+One", sans-serif;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.beige2 {
+  background-color: #fffaf8;
+  height: 45px;
+}
+
+.funding-grid-item4 {
+  display: none;
+}
```

Comment:
Same style definition of the selector on line 133. Group selectors or create a new class.